packages/migration/updates/m120.go

```diff
@@ -30,7 +30,7 @@ package updates
 
 var M120 = `
 ALTER TABLE "1_history" ADD COLUMN "created_new" bigint NOT NULL DEFAULT '0';
-UPDATE "1_history" SET created_new = EXTRACT(EPOCH FROM created_at::timestamp with time zone);
+UPDATE "1_history" SET created_new = EXTRACT(EPOCH FROM created_at::timestamp with time zone) WHERE created_at IS NOT NULL;
```

Comment:
you can't modify existing migration, because it never will be executed. 
You should define new one

Reply:
If this  migration has succesfully finished then this fix is not required.
But if some node has not yet executed this migration than this fix help it if this node have null time values in created_at. Otherwise, in this case, the migration will be failure.